Filter/ClassFilter.php

```diff
@@ -13,10 +13,11 @@
 
 use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
 use Sonata\CoreBundle\Form\Type\EqualType;
-use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
 
 class ClassFilter extends Filter
 {
+    const OPTION_SUB_CLASSES = 'sub_classes';
```

Comment:
To be clear, what's not great about this is that the const will no longer be needed in next major when we simplify this, but removing it will be a BC-break. Removing a local variable isn't.